src/v0/destinations/google_adwords_offline_conversions/networkHandler.js

```diff
@@ -27,14 +27,6 @@ const conversionCustomVariableCache = new Cache(CONVERSION_CUSTOM_VARIABLE_CACHE
 
 const createJob = async ({ endpoint, headers, payload, metadata }) => {
   const endPoint = `${endpoint}:create`;
```

Comment:
I see request & response logs removed here but not added in `httpPost` utility - what's the reason of this change ?

Reply:
Ignore, I see in network.js